tests/network/ping.pm

```diff
@@ -0,0 +1,41 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: iputils
+# Summary: Ping another machine
```

Comment:
I suggest to mention the purpose for the custom size

Reply:
Updated Summary.